Dizzy/Screens/SignIn/SignInDetailsView.swift

```diff
@@ -81,7 +81,7 @@ final class SignInDetailsView: UIView {
     private func layoutSignInButton() {
         stackView.addArrangedSubview(signInButton)
         signInButton.snp.makeConstraints { (signInButton ) in
-            signInButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage)
+            signInButton.width.equalToSuperview().multipliedBy(buttonsWidthPrecentage * 0.6)
```

Comment:
same as my note in `SingleDetailsView` dont use `*`, create constant called `signInButtonWidthPrecentage` or similar name and use it